server_addon/maya/client/ayon_maya/plugins/publish/collect_yeti_rig.py

```diff
@@ -271,22 +272,25 @@ def get_sequence(self, filepath, pattern="%04d"):
             pattern (str): The pattern to swap with the variable frame number.
 
         Returns:
-            list: file sequence.
+            list[str]: file sequence.
```

Comment:
We may return `None` if no files exist on disk that match.
```suggestion
            Optional[list[str]]: file sequence.
```